app/views.py

```diff
@@ -238,6 +108,31 @@ def events_list():
     return results, status.HTTP_200_OK
 
 
+@app.route("/api/events/search/", methods=['GET', 'POST'])
+def filter_or_search_events():
+    """Search or Filter the events list"""
+    page = request.args.get('page', 1, type=int)
+    category = request.args.get('category')
+    location = request.args.get('location')
+    if category:
```

Comment:
nice Implementation of filtering the events by category and location, although what if I want to filter for an event by both the location and category, will this be possible?

Reply:
as per now it is possible, I will refactor that

Reply:
@meshnesh @frankip , since there are only two categories, to filter an event by both category and location, is essentially displaying all the results in the db. Though it's definitely, a useful feature if you have more than two categories